spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
```

Comment:
Sorry, I know I've suggested this before, but can we please rename this one further to just `external`? It'll be nicer to see `external:ubuntu-core*` on the output than `external-ubuntu-core:ubuntu-core`.